features/generate_report.feature

```diff
@@ -20,6 +20,7 @@ Feature: Report Generation
           When I give dog food to the dog
           Then the dog will eat it
 
+        # TODO: add examples for left to right and right to left petting directions

```

Comment:
So, probably would be best if this was not a TODO.  At first glace I thought it was a real  TODO and not just an example comment.....

```suggestion
        # Do not need examples for left to right and right to left petting directions
```

Reply:
Fixed.